webapp/public/js/domjudge.js

```diff
@@ -80,7 +86,9 @@ function sendNotification(title, options = {})
     var not = new Notification(title, options);
 
     if ( link!==null ) {
-        not.onclick = function() { window.open(link); };
+        not.onclick = function() { 
+            window.location.href = link;
```

Comment:
What is the reason this has been changed? I think it should both work but maybe you had a reason it didn't?

Reply:
`window.open` will open the link in a new tab but won't switch to it, which might be confusing, so I replace that to open the link just in current tab.

Reply:
Hmmm I think that does make more sense yes.